librz/core/hack.c

```diff
@@ -267,7 +267,7 @@ RZ_API bool rz_core_hack_x86(RzCore *core, const char *op, const RzAnalysisOp *a
 	return true;
 }
 
-RZ_API int rz_core_hack(RzCore *core, const char *op) {
+RZ_API bool rz_core_hack(RzCore *core, const char *op) {
```

Comment:
Add Doxygen here please, especially since it's completely unclear from the name of the function what it is actually is doing.